includes/abstracts/abstract-wc-stripe-payment-gateway.php

```diff
@@ -901,20 +903,35 @@ public function process_refund( $order_id, $amount = null, $reason = '' ) {
 			return $response;
 
 		} elseif ( ! empty( $response->id ) ) {
-			$order->update_meta_data( '_stripe_refund_id', $response->id );
-
-			$amount = wc_price( $response->amount / 100 );
-
+			$formatted_amount = wc_price( $response->amount / 100 );
 			if ( in_array( strtolower( $order->get_currency() ), WC_Stripe_Helper::no_decimal_currencies() ) ) {
-				$amount = wc_price( $response->amount );
+				$formatted_amount = wc_price( $response->amount );
+			}
+
+			// If charge wasn't captured, skip creating a refund and cancel order.
+			if ( 'yes' !== $captured ) {
+				/* translators: dollar amount */
```

Comment:
Maybe replace `dollar amount` with `1) amount (including currency symbol)` as the amount might be in another currency

Reply:
I agree your suggestion sounds better. I copied `dollar amount` from another translation in the same file. I'm gonna update that one too.